.github/workflows/python-release.yml

```diff
@@ -9,18 +9,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
         python-version: '3.x'
-    - name: Install pypa/build
+
+    - name: Install poetry
       run: |
-        python -m pip install build --user
+        python -m pip install poetry
```

Comment:
poetry recommends installation through pipx in CI pipelines: https://python-poetry.org/docs/#ci-recommendations

@ldruschk we probably don't want to pin a version since we'd like to fail as soon as possible, right?

Reply:
I actually wanted to use `install build --user` to test: https://github.com/enowars/enochecker_test/blob/427240c6f7182f596cc27e3fdc3a39c67504bf90/pyproject.toml#L53-L55

Reply:
Not sure if I understand that correctly. Do you want to install the packages defined in the requires section using a specific pip command? Would be nice, but it does not appear to be possible

Reply:
nvm, I misread the changes and comment. It is correct as commited.

But about pinning the poetry version: I am not sure if dependabot is able to update the version here, so I would always use the latest version